src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -1,4 +1,27 @@
 package com.kodilla.ecommercee.controller;
 
+import com.kodilla.ecommercee.entity.GenericEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "/users", produces = "application/json")
 public class UserController {
+    @PostMapping("/createUser")
+    public void createUser(@RequestBody GenericEntity userDto){
+        new GenericEntity();
+    }
+
+    @GetMapping("/{id}")
+    public GenericEntity getUser(@RequestParam Long userId){
+        return new GenericEntity();
+    }
+
+    @PutMapping("/{id}/update")
```

Comment:
po co jest parametr id w url skoro przesyłasz id w json

Reply:
Otóż... taka była sugestia pozostałych członków zespołu. Mój pierwotny pomysł był bez id.

Reply:
różnica jest taka:
PathVariable: `/getSomething/1`
RequestParam: `/getSomething?somethingId=1`

które jest według Ciebie lepsze ?

Reply:
ale czy w przypadku update to też ma sens? bo muszę przesłać json, żeby zrobić update

Reply:
Czyli w przypadku update lepiej jest pozostawić normalnie ścieżkę "/delete"? bo i w tak w json jest zawarte id